models.js

```diff
@@ -1,9 +1,22 @@
 const db = require('./db/connection')
+const fs = require('fs')
 
 function getTopics(){
     return db.query("SELECT * FROM topics").then((result) =>{
         return result;
     })
 }
 
-module.exports={getTopics}
\ No newline at end of file
+function getAllEndPoints() {
+    return new Promise((resolve, reject) => {
+        fs.readFile('endpoints.json', 'utf-8', (err, data) => {
+            if (err) {
```

Comment:
We don't need this unless you have an error test - remember that all code should be test driven. We have only tested for the happy path so don't need any error handling logic (and don't need to worry about sad path here because we don't have anything coming from the user).